wp-resources/plugins/sil-dictionary-webonary/include/configuration.php

```diff
@@ -229,14 +229,13 @@ function save_configurations()
 		$useCloudBackend = filter_input(
 			INPUT_POST,
 			'useCloudBackend',
-			FILTER_SANITIZE_STRING,
+			FILTER_UNSAFE_RAW,
```

Comment:
Is this change from FILTER_SANITIZE_STRING to FILTER_UNSAFE_RAW on purpose? Here and below. Seems suspicious that we are no longer sanitizing.